ResearchUXFactory/SBAInfoManager.h

```diff
@@ -68,7 +68,24 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, copy) NSString * _Nullable keychainAccessGroup;
 
 /**
- App group identifier used for the suite name of NSUserDefaults (if provided).
+ Whether to use standard user defaults as the fallback.
+ For backward compatibility only, when upgrading an app originally built with old
+ Bridge frameworks that used standard user defaults. Has no effect if either userDefaultsSuite
+ or appGroupIdentifier is set.
+ 
+ If neither of those is set, and this is not set or is set to NO, then the default internal suite
+ for Bridge frameworks will be used (org.sagebase.Bridge).
```

Comment:
Apple standard is to use `group.org.sagebase.Bridge` instead. Also, does this still work if the user doesn't have app groups enabled?